README.md

```diff
@@ -72,6 +72,8 @@ A curated list of awesome research papers, datasets and software projects devote
 
 #### Program Synthesis and Induction
 
+* <img src="badges/12-pages-beginner-brightgreen.svg" alt="11-pages" align="top"> [Compiler Fuzzing through Deep Learning](https://chriscummins.cc/pub/2018-issta.pdf) - Chris Cummins, Pavlos Petoumenos, Alastair Murray, Hugh Leather, ISSTA 2018
```

Comment:
This one would better fit the [Program Repair and Bug Detection](https://github.com/src-d/awesome-machine-learning-on-source-code#program-repair-and-bug-detection) section :)

Also:
```suggestion
* <img src="badges/11-pages-beginner-brightgreen.svg" alt="11-pages" align="top"> [Compiler Fuzzing through Deep Learning](https://chriscummins.cc/pub/2018-issta.pdf) - Chris Cummins, Pavlos Petoumenos, Alastair Murray, Hugh Leather, ISSTA 2018
```